internal/api/thumbnails.go

```diff
@@ -2,10 +2,12 @@ package api
 
 import (
 	"fmt"
-	"github.com/gin-gonic/gin"
-	"github.com/photoprism/photoprism/internal/photoprism"
 	"log"
 	"strconv"
+
```

Comment:
Please run "make fmt" before pushing, it should order the imports alphabetically